### ISSUE TITLE
Guppy multiple server ports

### ISSUE THREAD

**Issue by esteinig** [nanoporetech/megalodon#28]
hi marcus, sorry to bug again. i noticed that it's currently not possible to run multiple forks of `guppy_basecall_server` conveniently within a nextflow on the same machine. 

the issue is that `--guppy-server-port` has to be specified manually and cannot be overwritten by e.g. `--guppy-params "--port auto"` . it also does not accept strings as argument e.g. `--guppy-server-port auto`

maybe a convenient fix for this would be to be able to pass `auto` to megalodon `--guppy-server-port`?

**Comment by esteinig** [comment]
to be fair it's fairly easily fixed by using a counter and add the port manually in the nextflow channel e.g.

```
port = 5555
Channel.fromPath("$params.dirs/*", type: 'dir')
        .map { 
           port += 1;
           tuple(port, it)
         }
```

**Comment by marcus1487** [comment]
Yes. I noticed this recently and have fixed this internally. In the next release `auto` will be the default. I’ll leave this issue open until the next release is pushed.

**Comment by marcus1487** [comment]
Fixed in 2.1 release.
